src/config.ts

```diff
@@ -297,8 +297,16 @@ export const defaultConfig: Config = {
     borders: false
   },
 
+  range: {
```

Comment:
No you shouldn't change the ramp config as this will change all examples and also affect the sequential scale's default.  

```
{
      scheme: 'blues',
      extent: [0, 1]
    }
```

is good though if you return it directly.

Reply:
Well, if we do that, we can use better colors.